crates/matrix-sdk/src/widget/capabilities.rs

```diff
@@ -73,6 +77,8 @@ const READ_EVENT: &str = "org.matrix.msc2762.receive.event";
 const SEND_STATE: &str = "org.matrix.msc2762.send.state_event";
 const READ_STATE: &str = "org.matrix.msc2762.receive.state_event";
 const REQUIRES_CLIENT: &str = "io.element.requires_client";
+pub const SEND_DELAYED_EVENT: &str = "org.matrix.msc4157.send.delayed_event";
+pub const UPDATE_DELAYED_EVENT: &str = "org.matrix.msc4157.update_delayed_event";
```

Comment:
Any reason why these are `pub` while the others are not?

Reply:
they are used in the mod.rs file for the widget error response printing.

Reply:
but a `pub (super)` is enough actually!